Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./Footer.css";
+
+const Footer = () => (
+	<footer>
+		<icon className="comment">
+			<i class="far fa-comment" />
```

Comment:
I love the font awesome icons! I wanted to make you aware of font awesome's official react component. The component can be tricky to use, so I do usually just use the cdn the way you have it here. I assume the component is more useful in situations where you are building large scale production apps but it's worth trying out https://fontawesome.com/how-to-use/on-the-web/using-with/react